doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -516,3 +522,123 @@ arguments used for configuration.  Here is a table describing these emulated dev
    * - ``wdt-i6300esb``
      - Emulated i6300ESB PCI Watch Dog Timer (WDT) Intel processors use to
        monitor User VMs.
+
+Launch Script
+*************
+
+Launch script is used to launch a User VM from the ServiceVM command line. It
+is generated by configurator according to several settings for a User VM. With
+launch script, user does not bother tweaking acrn-dm parameters.
```

Comment:
Should we map the ACRN Configurator settings to the launch script / DM options? Would that be useful to users? Thoughts @gvancuts @dbkinder?

Reply:
> Should we map the ACRN Configurator settings to the launch script / DM options? Would that be useful to users? Thoughts @gvancuts @dbkinder?

Yes, but we could update with this information after the 3.0 release (I think).  For example, we can note that the ``-m, --memsize <memory_size>`` parameter gets its value from the post-launched VM "Memory Size" basic option. (Or vice versa, we include in the "Memory Size" option description that this sets the -memsize acrn-dm parameter).  Or both. More discussion.

Reply:
Agree that it can be a future enhancement.